### DISCUSSION TITLE
Passing generation data through Extras upscale.

### DISCUSSION THREAD

**Discussion by Vagrin** [lllyasviel/stable-diffusion-webui-forge#1726] [Q&A] [answered]
Anyone can answer this. In short: What setting do I turn on so when I upscale something in the Extras tab the result has all the initial generation data in it. I can't find it.
I select the best of my image generations and upscale them in Extras X2. This leaves me with a directory full of my best stuff. When I want to start from a past great image I drag it into PNG Info and go from there. I've turned on pass info in the past but can't find the correct setting to change. Thanks in advance.

**Comment by Liva24** [comment, marked as answer]
Hey man, apparently it is a bug i have the same problem and i made a post about it a few weeks ago #1204 
3 days after that i found this #1312  they were adressing the problem,  but i dont know what happend i still having the same problem even after a clean installation

**Reply by Vagrin** [reply]
Thanks. At least I know I'm not missing a setting somewhere. We'll get there! Onward and Upward.
